File: engine/model/src/main/scala/tools/packet/pyramid/PyramidScore.scala
package tools.packet.pyramid

import net.opcodes.SendOpcode
import tools.packet.PacketInput

class PyramidScore(private var _score: Byte, private var _exp: Int) extends PacketInput {
  def score: Byte = _score

  def exp: Int = _exp

  override def opcode(): SendOpcode = SendOpcode.PYRAMID_SCORE
}

File: engine/engine-model/src/main/java/tools/packet/pyramid/PyramidScore.java
package tools.packet.pyramid;

import net.opcodes.SendOpcode;
import tools.packet.PacketInput;

public record PyramidScore(Byte score, Integer exp) implements PacketInput {
   @Override
   public SendOpcode opcode() {
      return SendOpcode.PYRAMID_SCORE;
   }
}